src/main/java/com/plugish/woominecraft/Util/RcHttp.java

```diff
@@ -19,11 +19,9 @@
 
 public class RcHttp {
 
-	public WooMinecraft plugin;
-
-	public RcHttp( WooMinecraft plugin ){
-		this.plugin = plugin;
-	}
+	private CloseableHttpResponse response;

```

Comment:
The Apache HTTP client is not `thread-safe` so I tried to keep all of these variables in one method, and not globalize them.